exporter/prometheusremotewriteexporter/README.md

```diff
@@ -100,6 +100,13 @@ Several helper files are leveraged to provide additional capabilities automatica
 - [TLS and mTLS settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md)
 - [Retry and timeout settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md), note that the exporter doesn't support `sending_queue` but provides `remote_write_queue`.
 
+### Feature gates
```

Comment:
@dashpole does something like this work? Or did you have different format / wording in mind.